docs/setup/settings.asciidoc

```diff
@@ -29,6 +29,27 @@ service that Kibana uses to display map tiles in tilemap visualizations. By defa
 `tilemap.options.attribution:`:: *Default: `"© [Elastic Tile Service](https://www.elastic.co/elastic-tile-service)"`* The map attribution string.
 `tilemap.options.subdomains:`:: An array of subdomains used by the tile service.
 Specify the position of the subdomain the URL with the token `{s}`.
+
+[[regionmap-settings]] `regionmap`:: configuration object to include additional vector layers for use in the <<regionmap,Region Map visualization>>.
+It consists of one or more _layer_ objects, each representing a pointer to an external vector file containing a _geojson FeatureCollection_. The file must be in the WGS84 coordinate reference, and only include polygons. 
```

Comment:
Is there anything external we can link to here for WGS84 coordinate reference? Such as https://en.wikipedia.org/wiki/World_Geodetic_System.  I'm not sure if it's allowed to link to external documentation within our own. Thoughts @debadair?